Tasks/Common/webdeployment-common-v2/ziputility.ts

```diff
@@ -9,15 +9,12 @@ var archiver = require('archiver');
 
 const deleteDir = (path: string) => tl.exist(path) && tl.rmRF(path);
 
-const extractWindowsZip = async (fromFile: string, toDir: string, usePowerShell?: boolean) => {
-
```

Comment:
please add your fix here as well https://github.com/microsoft/azure-pipelines-tasks/blob/master/common-npm-packages/webdeployment-common-v2/ziputility.ts and update package version

Reply:
will take care once this is merged in master